src/api/constants/XLSX-CONFIG/INDEXES/DECLARATIONS/index.test.ts

```diff
@@ -0,0 +1,124 @@
+import DECLARATIONS_INDEXES, { DEFAULT_INDEXES } from '.';
+import DECLARATIONS_FIELD_IDS from '../../../field-ids/insurance/declarations';
+import { mockApplication } from '../../../../test-mocks';
+
+const {
+  MODERN_SLAVERY: {
+    CONDITIONAL_REASONS: { CANNOT_ADHERE_TO_ALL_REQUIREMENTS, OFFENSES_OR_INVESTIGATIONS, AWARE_OF_EXISTING_SLAVERY },
+  },
+} = DECLARATIONS_FIELD_IDS;
+
+describe('api/constants/XLSX-CONFIG/INDEXES/DECLARATIONS', () => {
+  describe('DEFAULT_INDEXES', () => {
+    it('should return an object with indexes', () => {
+      const expected = {
+        CANNOT_ADHERE_TO_ALL_REQUIREMENTS: 0,
+        OFFENSES_OR_INVESTIGATIONS: 0,
+        AWARE_OF_EXISTING_SLAVERY: 0,
+      };
+
+      expect(DEFAULT_INDEXES()).toEqual(expected);
```

Comment:
```suggestion
      const result = DEFAULT_INDEXES();
      expect(result).toEqual(expected);
```

Reply:
Good shout, have updated all instances of this.